CONTRIBUTING.md

```diff
@@ -0,0 +1,190 @@
+make docker-build docker-push IMG=<some-registry>/kogito-serverless-operator:tag
+```
+
+3. Deploy the controller to the cluster with the image specified by `IMG`:
+
+```sh
+make deploy IMG=<some-registry>/kogito-serverless-operator:tag
+```
+
+### Uninstall CRDs
+To delete the CRDs from the cluster:
+
+```sh
+make uninstall
+```
+
+### Undeploy controller
+UnDeploy the controller to the cluster:
+
+```sh
+make undeploy
+```
+
+Minikube or CRC
+
+## Test the Greeting workflow on Minikube
+
+A good starting point to check that everything is working well, it is the [Greeting workflow](https://github.com/kiegroup/kogito-examples/blob/stable/README.md#serverless-workflow-getting-started).
+
+Follow these steps to create a container that you can than deploy as a Service on Kubernetes or KNative.
+
+1. Start Minikube
+```sh 
+minikube start --cpus 4 --memory 4096 --addons registry --insecure-registry "10.0.0.0/24"
+```
+2. Create a namespace for the building phase
+
+```sh
+kubectl create namespace kogito-builder
+```
+
+3. Create a secret
+```sh
+kubectl create secret docker-registry regcred --docker-server=<registry_url> --docker-username=<registry_username> --docker-password=<registry_password> --docker-email=<registry_email> -n kogito-builder
+```
+
+4. Build and push your image to the location specified by `IMG`:
+
+```sh
+make container-build container-push IMG=<some-registry>/kogito-serverless-operator:tag
+```
+
+5. Deploy the controller to the cluster with the image specified by `IMG`:
+
+```sh
+make deploy IMG=<some-registry>/kogito-serverless-operator:tag
+```
+
+6. Create a dedicated Namespace for the test:
+
+```sh
+kubectl create namespace greeting-workflow
+```
+
+7. Install Instances of Custom Resources:
+
+```sh
+kubectl apply -f config/samples/sw.kogito.kie.org__v08_kogitoserverlessworkflow.yaml -n greeting-workflow
+```
+
+8. Default configuration
+
+A configmap called kogito-serverless-operator-builder-config will be created under the kogito-serverless-operator-system
+namespace when the Operator will be installer, and it contains:
+
+- DEFAULT_BUILDER_RESOURCE = kogito_builder_dockerfile.yaml
+- DEFAULT_WORKFLOW_DEXTENSION = .sw.json
+- DEFAULT_KANIKO_SECRET_DEFAULT = regcred
+- DEFAULT_REGISTRY_REPO = quay.io/kiegroup
+- kogito_builder_dockerfile.yaml = dockerfile content
+
+For the local development the DEFAULT_REGISTRY_REPO must be changed
+
+# Customize Builder Image
+At the startup a 
+[Dockerfile](./config/manager/kogito_builder_dockerfile.yaml) 
+is placed in a configmap
+
+This Dockerfile use a base image called [swfbuilder](https://github.com/kiegroup/kogito-images/tree/master/modules/kogito-swf-builder) with installed:
+
+- openjdk 11
+- maven
+
+A Quarkus project  /home/kogito/serverless-workflow-project with the extensions
+- quarkus-kubernetes 
+- kogito-quarkus-serverless-workflow 
+- kogito-addons-quarkus-knative-eventing
+
+all the dependencies needed in the /home/kogito/.m2
+additional scripts in case of need to: 
```

Comment:
in case of need to, what exactly?
can you please ellaborate?

Reply:
ok